documentation/markup.rst

```diff
@@ -429,7 +432,7 @@ following example shows all of the features of this directive type::
                   ham(eggs)
        :noindex:
 
-       Spam or ham the foo.
+       Spam or ham the eggs.
```

Comment:
Please revert unrelated changes.

```suggestion
       Spam or ham the foo.
```

Reply:
Alright, let's leave it then, it's too minor for another pull request.